docs/source/get_started/installation.rst

```diff
@@ -57,8 +44,65 @@ using their OS package manager.
 .. _Python Downloads: http://python.org/download
 
 
+Enaml and Dependencies
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can install enaml and (almost) all of its dependencies through the pip
+command line.::
+
+    $ pip install enaml
+
+Toolkit Rendering Library
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enaml's declarative widgets provide a layer of abstraction on top of the
+widgets of a toolkit rendering library. You will need to install this
+dependency separately.
+
+Enaml supports using either PyQt4 or PyQt5 and uses the `QtPy`_ library
+as compatibility layer.
+
+The recommended library is `PyQt5`_,  a robust set of Python bindings to the
+Qt 5 toolkit.  (It includes the necessary parts of Qt 5.)
+
+On 32 and 64-bit Windows, 64-bit OS X and 64-bit Linux, with Python 
+versions >=3.5, it can be installed via pip::
```

Comment:
It might be good to explicitly note that pip doesn't have a wheel for 32-bit linux. I thought this wouldn't be an issue but had several users have problems with this because they still had 32 bit linux

Reply:
Fixed in next commit.

I followed the existing documents usage of `note::`, which looks pretty ordinary in GitHub's RST (and they WONTFIX) but I am hoping looks better in ReadTheDocs.